exercises/static/exercises/drone_cat_mouse/web-template/hal_guest.py

```diff
@@ -13,7 +13,7 @@ def __init__(self):
         rospy.init_node("HAL_mouse")
 
         self.image = None
-        self.mouse = DroneWrapper(name="rqt", ns="mouse/")
```

Comment:
undo all changes in this file when drone_wrapper backend is ready

Reply:
sure